homeassistant/components/suez_water/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+  "domain": "suez_water",
+  "name": "Compteur d'eau Suez",
+  "documentation": "https://github.com/ooii/home-assistant/tree/suezWater/homeassistant/components/suez_water",
+  "dependencies": [],
+  "codeowners": [],
+  "requirements": ["requests==2.21.0", "regex==2019.4.14","voluptuous==0.11.5", "datetime==4.3", "pysuez==0.1.11"]
```

Comment:
`pysuez` has [dependencies](https://github.com/ooii/pySuez/blob/master/setup.py#L33). Don't list the requirements which are ["core" dependencies](https://github.com/home-assistant/home-assistant/blob/dev/setup.py) and already pulled in by the module you are using.